### PR TITLE
feat: image type on table cell, added

### PR DESCRIPTION
**Pull request by saxenashivang**
Some use case would be to display an image in a table cell.

### CODE REVIEW

File: frontend/src/Editor/Components/Table/Table.jsx [hunk 1]
```diff
@@ -522,6 +524,23 @@ export function Table({
...
+            return (
+              <div>
+                <img
+                  src={cellValue}
+                  className={`rounded-circle`}
+                  style={{
+                    objectFit: 'contain',
+                    pointerEvents: 'auto',
+                  }}
+                  onClick={() => fireEvent('onClick')}
```
Comment (kavinvenkatachalam) [inline frontend/src/Editor/Components/Table/Table.jsx:537]: @sherfin94 Can you check this? Do we need `onClick` event for the image?

**Review by sherfin94 — CHANGES_REQUESTED** [review]
Hey @saxenashivang , thank you so much for the PR!

Presently, the image border seems to be set to rounded shape, would it be possible for you to make it customisable? That is, with different border types similar to that of the Image widget. You can edit how the column configuration popover on file `frontend/src/Editor/Inspector/Components/Table.jsx`

image

**Comment by withshubh** [comment]
@saxenashivang - Please make the required changes in the PR.

**Comment by withshubh** [comment]
Hey @saxenashivang 👋 
I hope you're doing good. Can you please update the PR with the suggested changes ?

**Comment by shubh22** [comment]
Task linked: [CU-2p7cv60 feat: image type on table cell, added #3417](https://app.clickup.com/t/2p7cv60)

**Comment by withshubh** [comment]
Closing the PR. ref:
- https://github.com/ToolJet/ToolJet/pull/4547

**Pull request closed without merging**